src/components/search/search.js

```diff
@@ -19,9 +20,28 @@ class SearchComponent extends React.Component {
     this.setState({ postcode: event.target.value });
   }
 
-  doPostcodeSearch = props => {
-    this.setState({ doSearch: true });
-  };
+  doPostcodeSearch() {
+    console.log(this.state.postcode);
+    if (this.state.postcode) {
+      let url =
+        "http://localhost:8000/api/venue/?postcode=" + this.state.postcode;
```

Comment:
I think we need to check what would the production url be as I believe we can't reference it as localhost even with a Docker setup

Reply:
I suggest that we get the url from an env file and this way we can have a different env file for production.